iotSql/parseSql.js

```diff
@@ -1,6 +1,6 @@
 const BASE64_PLACEHOLDER = '*b64'
 const SQL_REGEX = /^SELECT (.*) FROM '([^']+)'/
-const SELECT_PART_REGEX = /^(.*?)(?: as (.*))?$/
+const SELECT_PART_REGEX = /^(.*?)(?: (as|AS) (.*))?$/
```

Comment:
Wouldn't it be enough just to add `i` flag to the end of the regex to make it case insensitive?